From: Masanari Iida <standby24x7@gmail.com>
Date: Fri, 3 Aug 2012 02:36:51 +0000
Subject: [PATCH 100/143] emulex: benet: Add a missing CR in the end of
 message
Patch-mainline: v3.6-rc2
Git-commit: f3f9f332e231abaf28e14344733b7d000b830f7c
References: bnc#777565 FATE#313819

Missing a CR in printk causes 2 messages printed in one line.

Signed-off-by: Masanari Iida <standby24x7@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/emulex/benet/be_main.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/emulex/benet/be_main.c
+++ b/drivers/net/ethernet/emulex/benet/be_main.c
@@ -1931,7 +1931,7 @@ static int be_rx_cqs_create(struct be_ad
 
 	if (adapter->num_rx_qs != MAX_RX_QS)
 		dev_info(&adapter->pdev->dev,
-			"Created only %d receive queues", adapter->num_rx_qs);
+			"Created only %d receive queues\n", adapter->num_rx_qs);
 
 	return 0;
 }
